tools/MemoryCheckstyle.xml

```diff
@@ -36,9 +36,17 @@ under the License.
   <property name="charset" value="UTF-8"/>
   <property name="severity" value="warning"/>
   <property name="fileExtensions" value="java"/>
+  <property name="basedir" value="${basedir}"/>
```

Comment:
why is this added? I seem to recall it caused issues with VS Code and we removed it from ds-java.

Reply:
Please remove this line and we can close out this PR

Reply:
Sorry 'bout that. I was trying to figure out how to resolve ${basedir} and it seems that the checkstyle plugin has to be declared in the pom in order for this to happen.
Then I forgot that that property was still in there.